packages/react-compose/src/next/getSlots.test.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { getSlots } from './getSlots';
+import { nullRender } from './nullRender';
+
+describe('getSlots', () => {
```

Comment:
Generally would be better in my opinion if we omit the `renders` prefix in the tests as this is just a utility that is not rendering anything. Then for the tests which are actually rendering something, we can test whether the result rendered is really what we would expect.

Reply:
not sure what i was thinking there. Good suggestion. fixed.